Dockerfile

```diff
@@ -8,7 +8,7 @@ LABEL Description="Tyk Dashboard docker image" Vendor="Tyk" Version=$TYKVERSION
 RUN apt-get update \
  && apt-get upgrade -y \
  && apt-get install -y --no-install-recommends \
-            curl ca-certificates apt-transport-https \
+            curl ca-certificates apt-transport-https gnupg npm \
             build-essential \
  && curl -L https://packagecloud.io/tyk/tyk-dashboard/gpgkey | apt-key add - \
  && curl -sL https://deb.nodesource.com/setup_8.x | bash - \
```

Comment:
I suggest updating this too, nodejs 8.x is way too old. Let's go with 10.x instead if it doesn't error out when installing aglio.